examples/compute/instance/front_worker/main.tf

```diff
@@ -0,0 +1,25 @@
+# example of vkcs_compute_instance for vkcs_lb_members
+resource "vkcs_compute_instance" "front_worker" {
```

Comment:
lets add the example to doc template after "multiple network" example

Reply:
add